File: 20180713/Longest Continuous Increasing Subsequence.java
class Solution {
    public int findLengthOfLCIS(int[] nums) {
        int res = 0;
        int idx = 0;
        for(int i = 1; i < nums.length; i++){
            if(nums[i] <= nums[i-1]){
                res = Math.max(res, i - idx);
                idx = i;
            }
        }
        return Math.max(res, nums.length - idx);
    }
}

File: 20180713/Longest Continuous Increasing Subsequence.py
class Solution(object):
    def findLengthOfLCIS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        res = 0
        idx = 0
        for i in range(1, len(nums)):
            if nums[i] <= nums[i-1]:
                res = max(res, i - idx)
                idx = i
        return max(res, len(nums) - idx)
        